### ISSUE TITLE
GPG sign on Windows?

### ISSUE THREAD

**Issue by hadrienbecle** [fork-dev/Tracker#1812]
Hey,

Sorry about the question, but I couldn't find how to sign commits with GPG on Windows.

I understand there should be a GPG tab in the settings repository, at least on macOS (screenshots in #20). Does this feature exist on Windows?

Thanks,

**Comment by DanPristupov** [comment]
No, there's no similar UI on Windows. Currently, you must configure GPG in the command line.

**Comment by hadrienbecle** [comment]
Okay, that's what I thought :/ I configured GPG and told Fork which git instance to use, and it works perfectly. Thanks for the (really fast) answer!

Is the feature on the roadmap?
